UserService/src/main/java/com/libraryManager/UserService/service/UserService.java

```diff
@@ -0,0 +1,9 @@
+package com.libraryManager.UserService.service;
+
+import com.libraryManager.UserService.dto.UserEntity;
+
+
+public interface UserService {
+
+    public UserEntity createUser(UserEntity userEntity);
```

Comment:
Return type for saving an entity should be of type void or sny other type but we must not return the entity as a whole here